library/ssl_tls.c

```diff
@@ -11221,8 +11220,6 @@ static int ssl_write_real( mbedtls_ssl_context *ssl,
 {
     int ret = mbedtls_ssl_get_max_out_record_payload( ssl );
     const size_t max_len = (size_t) ret;
-    volatile const unsigned char *buf_dup = buf;
```

Comment:
It would be great to have stuff removed as a separate commit, but that's just a suggestion.

Reply:
It is not needed because I added this buffer data to the context , therefore I think it may be a part of the same commit